src/mmw/js/src/modeling/templates/scenarioTabPanel.html

```diff
@@ -6,7 +6,7 @@
             <i class="fa fa-caret-down"></i>
             </button>
             <ul class="dropdown-menu menu-right" role="menu">
-                {% if editable %}
+                {% if editable and not is_new %}
```

Comment:
Perhaps the conditional hierarchy in this page can be simplified? Unrelated to your changes, but a good opportunity to simplify. Currently we have:

```
if editable and not is_new
    Share
endif
Print
if not is_current_conditions
    if editable
        Rename
        Duplicate
        Delete
    endif
endif
```

Which might be simplified to:

```
if editable
    if not is_new
        Share
    endif
    if not is_current_conditions
        Rename
        Duplicate
        Delete
    endif
endif
Print
```

However, this would change the order of the items, which might not be desirable at this time.